maps-ktx/src/test/java/com/google/maps/android/ktx/model/PolygonOptionsTest.kt

```diff
@@ -26,12 +26,12 @@ class PolygonOptionsTest {
     @Test
     fun testBuilder() {
         val polygonOptions =
-            com.google.maps.android.ktx.buildPolygonOptions {
+            polygonOptions {
                 strokeWidth(1.0f)
                 strokeColor(Color.BLACK)
                 add(LatLng(0.0, 0.0))
```

Comment:
If would be safer if these values weren't `0.0`, given that it's the default value for `double`